src/clone/clone-utils.php

```diff
@@ -0,0 +1,179 @@
+<?php
+
+namespace EE\Site\Cloner\Utils;
+
+use EE;
+use EE\Site\Cloner\Site;
+use function EE\Utils\get_flag_value;
+use function EE\Utils\random_password;
+use function EE\Utils\trailingslashit;
+
+function copy_site_db( Site $source, Site $destination ) {
+
+	$site_type = $source->site_details['site_type'];
+	$db_host   = $source->site_details['db_host'];
+
+	if ( 'wp' === $site_type || 'php' === $site_type && ! empty( $db_host ) ) {
+		$source_site_name      = $source->site_details['site_url'];
+		$destination_site_name = $destination->site_details['site_url'];
+
+		EE::log( 'Exporting database from source' );
+
+		$filename       = $source_site_name . '-' . random_password() . '.sql';
+		$export_command = 'ee shell --skip-tty ' . $source_site_name . ' --command=\'wp db export ../' . $filename . '\'';
```

Comment:
This will not work if site type is `php`.

Reply:
Piyush was working on it, but last time we talked about it, @dhsathiya suggested skipping PHP site support for now. Do we really need it in this release?